static/js/template/services/template.service.js

```diff
@@ -80,17 +80,25 @@
                     icon: 'text_fields',
                     type: 'text',
                     sub_type: 'text',
+                    pattern_type: 'email',
```

Comment:
the properties on level 0 are the ones which every component has, so better to move this to `aux_attributes` as that's the object which has an unspecified structure and the property may or may not exist